app/views/shared/code/_extract_method_5.html.erb

```diff
@@ -0,0 +1,17 @@
+def print_owing
+  outstanding = 0.0
+
+  # print banner
+  puts "*************************"
+  puts "***** Customer Owes *****"
+  puts "*************************"
+
+  # calculate outstanding
+  @orders.each do |order|
+    outstanding += order.amount
+  end
+
+  # print details
+  puts "name: #{@name}"
+  puts "amount: #{outstanding}"
+end
```

Comment:
views in shared/code/ contain the code to refactor whereas views in shared/solution contain
the expected solution. But in any case, always views.